hardware/lock_in_amplifiers.py

```diff
@@ -18,7 +18,54 @@
 """
 
 import visa
+from hardware import u
+import random
 
+class MockLockInAmplifier:
+
+    def __init__(self, instr_name = None):
+        if not instr_name:
+            self.name = "Lock In Amplifier - SRS_SR844"
+        else:
+            self.name = instr_name
+
+        self._sensitivity_dict = {
+                0: {"Vrms": 100e-9, "dBm": -127},
+                1: {"Vrms": 300e-9, "dBm": -117},
+                2: {"Vrms": 1e-6, "dBm": -107},
+                3: {"Vrms": 3e-6, "dBm": -97},
+                4: {"Vrms": 10e-6, "dBm": -87},
+                5: {"Vrms": 30e-6, "dBm": -77},
+            }
+
+            # see page 115 of manual
+        self._time_constant_list = [  # query by index
+                100e-6,
+                300e-6,
+                1e-3,
+                3e-3,
+                10e-3,
+            ]
+
+
+    def identify(self):
+        return self.name
+
+    @property

```

Comment:
Set phase to be random when we call `__init__`, and then consistently return the same phase.